ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -293,6 +293,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "hearingDetails",
```

Comment:
I don't like hearingDetails, it's too generic. What do you think of `hearingBooking`?

Reply:
then it would be the same name as the complex type ? @Jamesnelson117

Reply:
True, what about hearingBookingDetails?

Reply:
@rebeccabrennan